docs/user-guide/tiramisu/finalizing-setup.md

```diff
@@ -11,8 +11,8 @@ We are going to make the Tiramisu environment start automatically when your cons
 1. Turn on your Wii U.
     - The Environment Loader should show up.
 1. Using the D-Pad, navigate to `tiramisu` and press Y to set this to your default envrionment, then press A to launch into Tiramisu.
-    - You might get a red warning screen telling you that updates aren't blocked properly. Press A to continue anyways. We're going to block updates in the "Blocking Updates" section below.
-    - To open the Environment Loader selector in the future, you have hold X while your Wii U is booting up.
+    - You might get a red warning screen telling you that updates aren't blocked properly. Press A to continue anyway. We're going to block updates in the "Blocking Updates" section below.
+    - To open the Environment Loader selector in the future, you have to hold X while your Wii U is booting up.
```

Comment:
While you're at it, can you change "Environment Loader selector" to simply "Environment Loader"?
```suggestion
    - To open the Environment Loader in the future, you have to hold X while your Wii U is booting up.
```